charts/ctsc-work-allocation/values.yaml

```
@@ -4,19 +4,29 @@ java:
   aadIdentityName: ctsc
   keyVaults:
     "ctsc":
-      resourceGroup: ctsc
       secrets:
-        - AppInsightsInstrumentationKey
-        - CTSC-IDAM-CLIENT-SECRET
-        - CTSC-SERVICE-USER-EMAIL
-        - CTSC-SERVICE-USER-PASSWORD
-        - CTSC-SERVICEBUS-CONNECTION-STRING
-        - wa-service-email-address
-        - wa-service-probate-email-address
-        - work-allocation-POSTGRES-PASS
-        - CTSC-WA-SMTP-USER
-        - CTSC-WA-SMTP-PASSWORD
-        - CTSC-S2S-SECRET
+        - name: AppInsightsInstrumentationKey
+          alias: ctsc.AppInsightsInstrumentationKey
```

I think it should be ?
```suggestion
          alias: AZURE_APPLICATIONINSIGHTS_INSTRUMENTATIONKEY
```

---

applied change